donate/templates/pages/core/landing_page_master.html

```diff
@@ -9,9 +9,9 @@
             <div class="image-feature column-spacing">
                 {% block featured_image %}
                     <div class="image-feature__container">
-                        {% image page.featured_image fill-290x95 as imageMobile %}
-                        {% image page.featured_image fill-690x183 as imageTablet %}
-                        {% image page.featured_image fill-350x197 as imageDesktop %}
+                        {% image page.featured_image width-290 as imageMobile %}
```

Comment:
`fill` resizes AND crops the image. Since we don't have control over what image user uploads and we don't want it to end up being cropped weirdly, I replaced `fill` with `width` to make sure the only treatment we apply is resizing.

Reply:
But wagtail has a "focal area" option for images, that lets people indicate which parts of an image "must" stay in view. Does that not work here?

![image](https://user-images.githubusercontent.com/177243/74772288-891f3900-5244-11ea-9fcf-bc9ba94a5705.png)

Reply:
I think the problem is we can’t predict what image CMS users upload. For this particular case, the image appears in different aspect ratios and it’s unlikely they will verify that the image they uploaded and cropped looks good on all screen sizes.

Reply:
current prod code
<img width="441" alt="image" src="https://user-images.githubusercontent.com/2896608/74781222-f2f40e80-5255-11ea-9eb7-ee9ce9e83c5f.png">